src/PrettierEditProvider.ts

```diff
@@ -22,6 +22,7 @@ import {
     ParserOption,
 } from './types.d';
 
+const bundledPrettier = require('prettier') as Prettier;
```

Comment:
I'm not that comfortable with typescript modules. This should be possible I think.
I would simply have rewritten it.